gf-webapp/src/logic/rooms/roomPage/roomPromo/RoomPromo.module.scss

```diff
@@ -19,33 +19,24 @@
 }
 
 .title {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  position: absolute;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  color: $color-white;
-  font-size: 60px;
-  font-family: Montserrat Regular;
+  @include room-h1;
 }
 
 .container,
 .services {
   background: $color-primary;
-  height: 50px;
+  min-height: 60px;
```

Comment:
Why height => min-height?

Reply:
### height: 50  
![Снимок экрана 2023-05-04 135752](https://user-images.githubusercontent.com/103531486/236158037-30bfdcb6-e0bc-47cd-b00a-7691e5a2d741.png)
### add fit-content  
![Снимок экрана 2023-05-04 135058](https://user-images.githubusercontent.com/103531486/236158166-8a868bcd-0e04-429c-abc6-7f28c84df6d0.png)
### looks better, but on a widescreen block takes the height of the string  
![Снимок экрана 2023-05-04 135114](https://user-images.githubusercontent.com/103531486/236158257-c440ddc6-15cd-4cc3-9f24-f63337450d5e.png)
### so, add min-height: 60. Because the block with services also has a height of 60px  
![Снимок экрана 2023-05-04 135141](https://user-images.githubusercontent.com/103531486/236158430-275b09e8-d92f-4375-8d28-d93e99d29732.png)